mache-chroniclemap/src/test/java/com/excelian/mache/chroniclemap/ChronicleMapMacheProvisionerShould.java

```diff
@@ -0,0 +1,88 @@
+package com.excelian.mache.chroniclemap;
+
+import com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.ChronicleMapBuilderConfig;
+import com.excelian.mache.core.HashMapCacheLoader;
+import net.openhft.chronicle.map.ChronicleMapBuilder;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.util.List;
+
+import static com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.chronicleMap;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.internal.util.reflection.Whitebox.getInternalState;
+
+public class ChronicleMapMacheProvisionerShould {
+
```

Comment:
Mate, I was the same initially about not calling my tests *Test when Mancuso threw it at us in his TDD workshop, but now when doing it (not that I did TDD with this stuff, clearly, my bad) using the *Should naming gets me in that mindset of "hey, what is this thing actually supposed to do..." It's the little things like that that I like about it.

Re underscores vs camel - I've moved on from the Osherove Underscore, it just stops the flow of reading. No longer a fan.

And Re the verb coming first - I play to my own personality there - I want to know what happens first, and then how we got there, I'm greedy like that. The name is describing the behaviour you're after, I don't care in which order you write it.

I completely agree that we need to standardise this though and I'm happy to go with whatever. Apart from the test internals I've seen around the project: I'm just not a fan of the named private method for each little chunk of the test - sorry JB I know you're a fan - I don't like having a bunch of instance vars and swapping back and forth between the test and a private method down south of the test class, understand what it does, fly back up to the test, and repeat for the next stages. Again though, I'm happily overruled on standards and can rewrite that way.